testing/test_physics_metrics.py

```diff
@@ -178,7 +179,9 @@ def clusterJets(pt, eta, phi, ptcut=0., deltaR=0.4):
     event = np.column_stack((pt_wptcut, eta_wptcut, phi_wptcut, mass_wptcut))
     event.dtype = DTYPE_PTEPM
     sequence = cluster(event, R=deltaR, p=-1)
-    jets = sequence.inclusive_jets(ptmin=30)
+    jets = sequence.inclusive_jets(ptmin=300)
```

Comment:
same here, ptmin should not be hard coded. add a parameter to the function and pass?